.gitignore

```diff
@@ -12,6 +12,10 @@
 /src/graphql/**/__generated__
 schema.graphql
 
+# generated ajv schema
+/src/utils/__generated__/*
+!/src/utils/__generated__/.gitkeep 
```

Comment:
nit: You can update this to exclude all generated files, which is desired, if you'd like:

```
# generated files
/src/**/__generated__

# schema
schema.graphql
```

Separately - why is the `.gitkeep` required here?

Reply:
was referencing widget package's .gitignore file. will see if it's really needed.